contracts/staking/src/types.ts

```diff
@@ -67,6 +67,62 @@ export class StoredBalance {
     ) {}
 }
 
+/**
+ * Simulates _loadCurrentBalance. `shouldMutate` flag specifies whether or not to update the given
+ * StoredBalance instance.
+ */
+export function loadCurrentBalance(
+    balance: StoredBalance,
+    epoch: BigNumber,
+    shouldMutate: boolean = false,
```

Comment:
Bit of a nit. Does this really make things cleaner? I mean if we didn't have it we'd just do a `balance = loadCurrentBalance(balance, ...)` vs `loadCurrentBalance(balance, ..., true)`. But the first seems clearer.